crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -262,6 +262,7 @@ def status
                 "pending" => 0,
                 "unready" => 0,
                 "building" => 0,
+                "crowbar-upgrade" => 0,
```

Comment:
It's not correct. It's INVALID, e.g. https://github.com/crowbar/crowbar-core/pull/177#discussion_r49061027
You can not use a dash within an object attribute.